release/lts/ubuntu20.04-arm32v7/docker/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation. 
```

Comment:
do we need LTS image for Ubuntu 20.04 arm32? We don't have an LTS image for Ubuntu 20.04 default architecture..